.logs/empty.xml

```diff
@@ -0,0 +1,3 @@
+<!-- An empty test suite which is needed as not all tasks produce XUnit results and the current Evergreen setup always expects some file to be uploaded -->
+<testsuite name="empty" tests="0" failures="0" errors="0" skipped="0" timestamp="Fri, 18 Oct 2024 09:21:06 GMT" time="8.907">
```

Comment:
I can't think of a better way to go about this... evergreen seems to fail the job completely when no files are found in the attach results step